velox/functions/prestosql/tests/MapFromEntriesTest.cpp

```diff
@@ -283,3 +284,115 @@ TEST_F(MapFromEntriesTest, rowsWithNullsNotPassedToCheckDuplicateKey) {
   ASSERT_NO_THROW(
       evaluate("try(map_from_entries(c0))", makeRowVector({arrayVector})));
 }
+
+TEST_F(MapFromEntriesTest, arrayOfDictionaryRowOfNulls) {
+  RowVectorPtr rowVector =
```

Comment:
nit: can you add a quick comment on what vector structure its testing, like

arrayOfDictionaryRowOfNulls: Array(DictionaryWithoutNulls(RowWithNulls(Flat1, Flat2)))
arrayOfConstantRowOfNulls: Array(ConstantWithoutNulls(RowWithNulls(Flat1, Flat2)))
arrayOfConstantNotNulls: Array(ConstantWithoutNulls(RowWithoutNulls(Flat1, Flat2)))